packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -339,29 +379,88 @@ class _RenderVectorGraphics extends RenderBox {
     return constraints.smallest;
   }
 
-  final Matrix4 transform = Matrix4.identity();
+  void invalidateRaster() {
+    _lastRasterizedSize = null;
```

Comment:
nit: consider just making this `markNeedsPaint()` and removing the `markNeedsPaint` above

Reply:
done